### PR TITLE
Feat(Builder): Enhance AITextSummarizerBlock with configurable summary style and focus

### PR DESCRIPTION
**Pull request by Torantulino**
### Background

This was requested by a couple of users and I've felt the need two when building agents.

### Changes 🏗️

The AITextSummarizerBlock in the autogpt_platform/backend/backend/blocks/llm.py file has been enhanced to include the following changes:
- Added a new enum class, SummaryStyle, with options for `concise`, `detailed`, `bullet points`, and `numbered list` styles.
- Added a new input parameter, `focus`, to specify the topic of the summary.
- Modified the _summarize_chunk method to include the style and focus in the prompt.
- Modified the _combine_summaries method to include the style and focus in the prompt.

These changes allow users to customize the style and focus of the generated summaries, providing more flexibility and control.

### CODE REVIEW

File: autogpt_platform/backend/backend/blocks/llm.py [hunk 1]
```diff
@@ -426,7 +434,7 @@ def llm_call(
         raise ValueError("Failed to get a response from the LLM.")
 
     def _summarize_chunk(self, chunk: str, input_data: Input) -> str:
-        prompt = f"Summarize the following text concisely:\n\n{chunk}"
+        prompt = f"Summarize the following text in a {input_data.style} form. Focus your summary on the topic of `{input_data.focus}` if present, otherwise just provide a general summary:\n\n```{chunk}```"
```
Comment (kcze) [inline autogpt_platform/backend/backend/blocks/llm.py:452]: Please run `poetry run format`. This line and long `prompt = ` below may still require breaking into multiple lines manually, so lint passes.

Comment (kcze) [inline autogpt_platform/backend/backend/blocks/llm.py:452]: Maybe 
```suggestion
        prompt = f"Summarize the following text in a {input_data.style} form. " + 
        (f"Focus your summary on the topic of `{input_data.focus}`" if focus else "") + 
        f"\n\n```{chunk}```"
```

**Review by kcze — CHANGES_REQUESTED** [review]

File: autogpt_platform/backend/backend/blocks/llm.py (hunk 1 above)
Comment (Torantulino) [reply autogpt_platform/backend/backend/blocks/llm.py:452]: With our new docker setup is there a new way to do this, or do I have to poetry install just to format?

**Review by kcze — APPROVED** [review]
LGTM

**Pull request merged**